.travis.yml

```diff
@@ -12,10 +12,25 @@ addons:
 cache:
   directories:
     - node_modules
-services:
-  #- docker
 before_script:
-  #Docker setup goes here
   - pwd
   - cp ~/build/HackIllinois/api-2017/config/test.config.template ~/build/HackIllinois/api-2017/config/test.config
```

Comment:
`before_script` runs before travis runs the unit tests which need the test env, and the production config should be copied over within the s3 bucket that the api is being deployed to